services/bigid-azure-key-vault-integration-app/src/test/java/com/bigid/azurekeyvaultapp/AzureKeyVaultApplicationTest.java

```diff
@@ -1,13 +1,13 @@
-package com.basicapp.basicdemoapp;
+package com.bigid.azurekeyvaultapp;
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
-class BasicDemoAppApplicationTests {
+class AzureKeyVaultApplicationTest {
 
-	@Test
-	void contextLoads() {
-	}
+    @Test
+    void contextLoads() {
```

Comment:
lets add unit tests

Reply:
> lets add unit tests

will add right after the doc and manual testing